src/design-system/stylesheets/components/_cookie-banner.scss

```diff
@@ -1,15 +1,15 @@
 .app-cookie-banner {
-  @include govuk-font-regular-16;
+  @include govuk-font(16, $weight: regular);
   @include govuk-text-colour;
 
   box-sizing: border-box;
   width: 100%;
 
-  padding-top: $govuk-spacing-scale-3;
-  padding-right: $govuk-spacing-scale-3;
-  padding-bottom: $govuk-spacing-scale-3;
-  padding-left: $govuk-spacing-scale-3;
-  background-color: $govuk-light-blue-25;
+  padding-top: govuk-spacing(3);
+  padding-right: govuk-spacing(3);
+  padding-bottom: govuk-spacing(3);
+  padding-left: govuk-spacing(3);
+  background-color: govuk-colour("light-blue")-25;
```

Comment:
extra -25?

Reply:
Somewhere we were actually using the tints 😱

Reply:
quick, let's get dave to suggest a colour :)

Reply:
I think lighten is relative, I think you'll need to use `scale-color` or `mix-color` and mix it with white. Colour manipulation isn't one of my strong points… guess check the result against the colour in master!

Reply:
lighten(desaturate(govuk-colour("light-blue"), 8.46), 42.55) :)